packages/directory-plugin/README.md

```diff
@@ -0,0 +1,30 @@
+# @wmr/ls-plugin
```

Comment:
This should probably be `directory-plugin`. There are a few other old references. I think we should stick with either one and be consistent in our naming 👍